plugins/dell-dock/README.md

```diff
@@ -192,3 +192,10 @@ This plugin has been available since fwupd version `1.2.0`.
       I2C~~~USB1
       I2C~~~USB2
 ```
+
+## Owners
+
+Anyone can submit a pull request to modify this plugin, but the following people should be
+consulted before making major or functional changes:
+
+* Mario Limonciello: @superm1
```

Comment:
Probably should add crag here, he's more in the know than me these days

Reply:
prefixed, thanks.